hypha/apply/activity/adapters/slack.py

```diff
@@ -65,6 +67,8 @@ class SlackAdapter(AdapterBase):
         MESSAGES.UPDATE_INVOICE: _('{user} has updated invoice for <{link}|{source.title}>'),
         MESSAGES.SUBMIT_REPORT: _('{user} has submitted a report for <{link}|{source.title}>'),
         MESSAGES.BATCH_DELETE_SUBMISSION: 'handle_batch_delete_submission',
+        MESSAGES.STAFF_ACCOUNT_CREATED: _('{user} has created a new account for <{link}|{source}>'),
+        MESSAGES.STAFF_ACCOUNT_EDITED: _('{user} has edited account for <{link}|{source}> that now has following roles: {roles}'),
```

Comment:
In the long run I would prefer to make these generic, in Markdown perhaps? Then each staff message adapter can translate it to whatever is needed.

Reply:
I agree, in terms of storage, I would prefer them to be JSON or JSON-serializable objects, compared to markdown.